server/plugin.go

```diff
@@ -318,7 +318,7 @@ func (p *Plugin) updateConfig(f func(conf *Config)) Config {
 }
 
 func (p *Plugin) GetPluginURLPath() string {
-	return "/plugins/" + manifest.ID + "/api/v1"
+	return "/plugins/" + manifest.Id + "/api/v1"
```

Comment:
Actually confluence master added some code in the build folder which was related to the creating of manifest file so due to that I copied the same code and used that to create the manifest file this change is because of that.